oxidation/crates/platform_web/Cargo.toml

```diff
@@ -3,4 +3,8 @@ name = "libparsec_platform_web"
 version = "0.0.0"
 edition = "2021"
 
+[[test]]
+name = "integration"
+path = "tests/tests.rs"
+
```

Comment:
not required, cargo will auto discover the test https://doc.rust-lang.org/cargo/reference/cargo-targets.html#target-auto-discovery.

but you may want to renamed the file to `tests/integration.rs`